app/controllers/profiles_controller.rb

```diff
@@ -18,7 +18,7 @@ def update
     @user = current_user.clone
     if @user.update(params_user)
       if @user.unconfirmed_email
-        Mailer.delay.email_reset(current_user)
+        send_email_reset_mails
```

Comment:
`send_email_reset_mails` is a bit confusing, maybe just `send_reset_emails`?

Reply:
yes, updated.